src/psyclone/psyGen.py

```diff
@@ -2750,8 +2772,14 @@ def apply(self, node, options=None):
         :type options: Optional[Dict[str, Any]]
 
         '''
+        # TODO 2668: options are now deprecated:
+        if options is not None:
+            print("Deprecation Warning: The options parameter to "
```

Comment:
Probably best to prefix this, e.g. "PSyclone deprecation warning: ..."

Reply:
Added to both.